### ISSUE TITLE
Bad looking losses

### ISSUE THREAD

**Issue by PierreBoyeau** [Naresh1318/Adversarial_Autoencoder#2]
Good Morning!

I am trying to reproduce the very nice results you have in your [tutorial](https://towardsdatascience.com/a-wizards-guide-to-adversarial-autoencoders-part-2-exploring-latent-space-with-adversarial-2d53a6f8a4f9) but my training do not go as well when I execute `python adversarial_autoencoder.py`

Here are the losses I obtain:
![image](https://user-images.githubusercontent.com/23222597/34719459-38a5a85a-f53b-11e7-9e6c-e36121401484.png)
I tried basically to decrease the learning rate without effect.
Can you help me?

Thanks in advance,

Pierre

**Comment by dmelladoc** [comment]
Have you modified anything besides the learning rate?
Are you running on GPU or CPU?

Because those two things might have had an effect on why you got those losses...

**Comment by Naresh1318** [comment]
Hey! Did you try looking at the generated images on Tensorboard?

**Comment by mszeto715** [comment]
Naresh, Thank you for writing your tutorial on AAEs. It's been very helpful. Unfortunately, I am running into the same problem as Pierre. See my scalar values below. Any insight you might have? Did you train for 1000 epochs or 300?

![screen shot 2018-05-24 at 11 34 15 am](https://user-images.githubusercontent.com/38442887/40495917-7b725d70-5f46-11e8-9818-b775fc71a4b5.png)
![screen shot 2018-05-24 at 11 34 11 am](https://user-images.githubusercontent.com/38442887/40495918-7b8279b2-5f46-11e8-91fb-4d8f5cef1a5a.png)
![screen shot 2018-05-24 at 11 34 07 am](https://user-images.githubusercontent.com/38442887/40495919-7b9ecf72-5f46-11e8-8e35-0b52f7719d18.png)

Thanks.
-Mimi
